pkg/services/notificator/nats/service.go

```diff
@@ -38,12 +38,12 @@ type Option func(*opts)
 var errConnIsClosed = errors.New("connection to the server is closed")
 
 // Notify sends object address's string representation to the provided topic.
-// Uses first 4 bytes of object ID as a message ID to support 'exactly once message'
-// delivery.
+// Uses first 4 bytes of object ID as a message ID to support 'exactly-once'
```

Comment:
It is exactly once QoS in NATS documentation.

Reply:
fixed